samples/fablo-config-hlf3-1orgs-1chaincode.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "https://github.com/hyperledger-labs/fablo/releases/download/2.0.0/schema.json",
+    "global": {
+      "fabricVersion": "3.0.0-beta",
+      "tls": true,
+      "monitoring": {
+        "loglevel": "debug"
+      }
+    },
+    "orgs": [
+      {
+        "organization": {
+          "name": "Orderer",
+          "domain": "orderer.example.com"
+        },
+        "orderers": [
+          {
+            "groupName": "group1",
+            "type": "raft",
+            "instances": 1
```

Comment:
please add at least 3 raft nodes, and also verify them in logs (we need to be sure all raft nodes are able to support the channel)